include/ctranslate2/ops/min_max.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "op.h"
+
+namespace ctranslate2 {
+  namespace ops {
+
+    class Min : public BinaryOp {
```

Comment:
Can you add small tests for the `Min` and `Max` operators in `tests/ops_test.cc`?